tools/gce_setup/build_images.sh

```diff
@@ -35,7 +35,7 @@ main() {
   cd ../../
 
   # build images for all languages
```

Comment:
let us not put this change in yet in case the test does not work right away.

adding it here will add it to nightly right away.